Algorithm/QCAlgorithm.cs

```diff
@@ -2499,9 +2499,11 @@ public bool RemoveOptionContract(Symbol symbol)
         /// open orders and then liquidate any existing holdings
         /// </summary>
         /// <param name="symbol">The symbol of the security to be removed</param>
+        /// <param name="tag">Custom tag to know who is calling this</param>
         [DocumentationAttribute(AddingData)]
-        public bool RemoveSecurity(Symbol symbol)
+        public bool RemoveSecurity(Symbol symbol, string tag = null)
         {
+            tag ??= "Removed Security";
```

Comment:
Minor but following the pattern of `Liquidate()` using `Liquidated` as default, I think the tag should be `Removed`